testcases/InstallUpstreamKernel.py

```diff
@@ -124,6 +125,7 @@ def is_url(path):
             log.debug("Compile and install linux kernel")
             con.run_command("make -j %d -s && make modules_install && make install" %
                             onlinecpus, timeout=self.host_cmd_timeout)
+            time.sleep(10)
```

Comment:
10 sec will work always ?

Reply:
@PraveenPenguin  Right Now 10 sec is working. If I add more time delay it will increase total execution time. So keeping this.  Please let me know your inputs.